src/main/java/com/accolite/pru/health/AuthApp/controller/AuthController.java

```diff
@@ -150,7 +150,7 @@ public ResponseEntity resetLink(@ApiParam(value = "The PasswordResetLinkRequest
 
         return authService.generatePasswordResetToken(passwordResetLinkRequest)
                 .map(passwordResetToken -> {
-                    UriComponentsBuilder urlBuilder = ServletUriComponentsBuilder.fromCurrentContextPath().path("/password/reset");
+                    UriComponentsBuilder urlBuilder = ServletUriComponentsBuilder.fromCurrentContextPath().path("/api/auth/password/reset");
```

Comment:
The base context path is automatically inherited from [here](https://github.com/isopropylcyanide/Jwt-Spring-Security-JPA/blob/23e371d3254b2fc0df8c2f700221741b3b1afbeb/src/main/java/com/accolite/pru/health/AuthApp/controller/AuthController.java#L60) and is hence not required.

Thanks for the contribution. However, this will will break the API. 

The API is using the context path already

```
curl --location --request POST 'localhost:9004/api/auth/password/resetlink' \
--header 'Content-Type: application/json' \
--data-raw '{
    "email": "a@b.com"
}'
```

Reply:
I build that logic  based on 
https://github.com/isopropylcyanide/Jwt-Spring-Security-JPA/blob/23e371d3254b2fc0df8c2f700221741b3b1afbeb/src/main/java/com/accolite/pru/health/AuthApp/controller/AuthController.java#L132

and 
https://github.com/isopropylcyanide/Jwt-Spring-Security-JPA/blob/23e371d3254b2fc0df8c2f700221741b3b1afbeb/src/main/java/com/accolite/pru/health/AuthApp/controller/AuthController.java#L213

where you did mention the base path.

Reply:
I see. Looks like there was some confusion. I see it now. Thanks for pointing it out.